ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -161,15 +161,53 @@ public void SetSource(Border view, JArray sources)
             }
         }
 
+        /// <summary>
+        /// Enum values correspond to positions of prop names in ReactPropGroup attribute
+        /// applied to <see cref="SetBorderRadius(Border, int, double)"/>
+        /// </summary>
+        private enum Radius
+        {
+            All,
+            TopLeft,
+            TopRight,
+            BottomLeft,
+            BottomRight,
+        }
+
         /// <summary>
         /// The border radius of the <see cref="ReactRootView"/>.
         /// </summary>
         /// <param name="view">The image view instance.</param>
+        /// <param name="index">The prop index.</param>
         /// <param name="radius">The border radius value.</param>
-        [ReactProp("borderRadius")]
-        public void SetBorderRadius(Border view, double radius)
+        [ReactPropGroup(
+            ViewProps.BorderRadius,
+            ViewProps.BorderTopLeftRadius,
+            ViewProps.BorderTopRightRadius,
+            ViewProps.BorderBottomLeftRadius,
+            ViewProps.BorderBottomRightRadius)]
+        public void SetBorderRadius(Border view, int index, double radius)
         {
-            view.CornerRadius = new CornerRadius(radius);
+            var cornerRadius = view.CornerRadius;
```

Comment:
>view.CornerRadius [](start = 31, length = 17)

Can `view.CornerRadius` ever be null? You can see `ReactViewManager` uses this expression:

```
var cornerRadius = border.CornerRadius == null ? new CornerRadius() : border.CornerRadius;
```